### DISCUSSION TITLE
Chainloading from https fails: Operation not permitted

### DISCUSSION THREAD

**Discussion by guettli** [ipxe/ipxe#1078] [General]
I try chainload via this URL:

https://thomas-guettler.de/tmp/alpine.ipxe

But I get this error:

>  Operastion not permitted not permitted

https://ipxe.org/err/410de1

![image](https://github.com/ipxe/ipxe/assets/414336/b52ca650-56c0-4ec9-99b2-2b751c63b220)

You can reproduce it with:

```
qemu-system-x86_64 -boot n -net nic -net user -m 512 -cdrom ipxe.iso
```

**Comment by NiKiZe** [comment]
Operation not permitted https://ipxe.org/410de13c
Here is part of that page:
> This error indicates that the server has sent an explicit fatal error.

> There should therefore be something in the server logs to indicates why the server has chosen to do this.

> Try to build with TLS [debug turned on](https://ipxe.org/download#debug_builds), and check for messages exchange between server and iPXE. If server replies with “02 28”, then TLS was [unable to negotiate compatible cipher](https://ipxe.org/crypto) suites between iPXE and your server. Enable something on the server side to show what is supported by iPXE.

Did you follow up on any of the suggestions given on the page?
I'd start with the last suggestion and verify that the server is configured to support any of the ciphers supported by iPXE.

I'd start with enabling earlier versions of TLS, https://www.ssllabs.com/ssltest/analyze.html?d=thomas-guettler.de
If you don't want to then there is always the option of allowing iPXE to use non https.
